### DISCUSSION TITLE
Welcome to react Discussions!

### DISCUSSION THREAD

**Discussion by simo-vargasp-sonarsource** [simo-vargasp-sonarsource/react#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by simo-vargasp-sonarsource** [comment]
ttt

**Reply by simo-vargasp-sonarsource** [reply]
test

**Reply by simo-vargasp-sonarsource** [reply]
test 2

**Comment by simo-vargasp-sonarsource** [comment]
eeee

**Comment by simo-vargasp-sonarsource** [comment]
rrrrr

**Comment by cloudglides** [comment]
whats redux
